Core/Core/Router/Router.swift

```diff
@@ -21,6 +21,7 @@ public protocol RouterProtocol {
     func route(to url: URL, from: UIViewController, options: Router.RouteOptions?)
     func route(to url: String, from: UIViewController, options: Router.RouteOptions?)
     func route(to url: URLComponents, from: UIViewController, options: Router.RouteOptions?)
+    func route(to view: UIViewController, from: UIViewController, options: Router.RouteOptions?)
```

Comment:
Since this was newly added to the protocol, does the Canvas router (`Canvas/Shrug/Router+Canvas.swift`) still conform?

Reply:
Oh good catch. I will make sure.